images/router/haproxy/conf/haproxy-config.template

```diff
@@ -151,6 +151,7 @@ backend be_tcp_{{$id}}
 
             {{ if eq $cfg.TLSTermination "reencrypt" }}
 backend be_secure_{{$id}}
+  mode http
```

Comment:
Yes, that solved all the issues that I was emailing about where I could write from server->client but couldn't read without an EOF.  I did a full manual regression test of the https routes with this setting too and everything checked out.  I then face palmed for approximately 35 seconds.